CONTRIBUTING.md

```diff
@@ -23,3 +23,52 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Setting up Bazel
+
+* Go to [http://bazel.build](http://bazel.build), click on `GET BAZEL`, and install Bazel.
+* Make sure you use Android Studio 3.2 or greater.
```

Comment:
I fear this won't age well. Would including something in the bazel docs about pre-3.2 not working in these docs be better: https://docs.bazel.build/versions/master/ide.html

Or maybe something like "Are you on Android Studio 3.1 or lower? See these instructions"

Reply:
Also, can we omit this hear and rely on the section you wrote below about Android Studio? It's not necessary for those that just want to build